news/test_squeeze.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Squeeze test: given a squeezed signal, unsqueezing recovers the original
```

Comment:
you don't need this.  This "news" is user facing documentation that will become the changelog at the next release.  It tells users how the code has changed since the last release.  Se we want to say something like, unded Added, * Polynomial squeeze of x-axis of morphed data" or something even more descriptive.